tensorflow-core/tensorflow-core-generator/pom.xml

```diff
@@ -28,15 +28,69 @@
       <artifactId>javapoet</artifactId>
       <version>1.12.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>${protobuf.version}</version>
+    </dependency>
     <dependency>
       <groupId>com.github.javaparser</groupId>
       <artifactId>javaparser-core</artifactId>
       <version>3.15.12</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.commonmark/commonmark -->
+    <dependency>
+      <groupId>org.commonmark</groupId>
+      <artifactId>commonmark</artifactId>
+      <version>0.17.1</version>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <!--
+          Add generated classes to the source path
+            These are classes generated by the javacpp-plugin, including TensorFlow operations
+            wrappers and C API Java bindings
+          -->
+          <execution>
+            <id>add-gen-sources</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>add-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>${project.basedir}/src/gen/protoc</source>
```

Comment:
I was under the impression that only the protos we need for generating the ops would be copied to the generator source tree (directly with other java sources, under `src/main/java`). AFAIK, there should be only a few needed.

We won't need then this `build-helper-maven-plugin`.

Reply:
I can do that, I just kept them in another folder for seperation.